apps/common/models/i-casework.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
-const Model = require('hof').model;
+const AuthToken = require('./auth-token');
 const crypto = require('crypto');
 
 const config = require('../../../config');
 
-module.exports = class CaseworkModel extends Model {
+module.exports = class CaseworkModel extends AuthToken {
```

Comment:
By extending off the auth model, it means this.auth() works in the Compose function above because any superclass declared there essentially are either this model or a descendant of it. If we used the HOF model directly, this.auth() doesn't do anything without a different setup.